frontend/src/components/ViewTrips/trip.js

```diff
@@ -30,19 +30,23 @@ export function getDateRange(tripData) {
 }
 
 /**
- * Return collaborator emails corresponding to the collaborator uid's
- * `collaboratorUidArr` in a comma separated string.
+ * Returns an array with the same contents as `collaboratorEmailArr` except that
+ * the current user email is moved to be the first element of the array (the
+ * other elements maintain their original order).
  *
- * @param {!Array<string>} collaboratorUidArr Array of collaborator uids
- *     stored in trip document.
- * @returns {string} Collaborator emails in comma separated string.
- *     Ex: "person1@email.com, person2@email.com".
+ * @param {!string[]} collaboratorEmailArr Array of user emails sorted in
+ *     alphabetical order.
+ * @return {!string[]} Array of user emails where first element is the current
+ *     user email and the following elements maintain their previous order.
  */
-export function getCollaboratorEmails(collaboratorUidArr) {
-  const collaboratorEmailArr = getUserEmailArrFromUserUidArr(collaboratorUidArr);
-  return collaboratorEmailArr.join(', ');
+export function moveCurUserEmailToFront(collaboratorEmailArr) {
+  collaboratorEmailArr = collaboratorEmailArr.filter(email => {
+    return email !== authUtils.getCurUserEmail();
```

Comment:
Based on what I have seen with the site rendering, no. Currently with the real-time updates (as of #96) everything seems to load pretty fast upon an edit except for the collaborators due to the asynchronous operation of converting user email to user uid and vice versa.

So while this should not be much of a concern, I will ask @keiffer01  to make sure.

Reply:
`getCurUserEmail` grabs details from the currently logged in user, which I store locally until the user logs out. Since we don't query the actual Firebase, there shouldn't be any significant overhead in calling this function many times.